examples/notebooks/parameter-management.ipynb

```diff
@@ -0,0 +1,373 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'a': 4,\n",
+       " 'b': 5,\n",
+       " 'c': 6}"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pybamm.ParameterValues(\"param_file.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Contributing a new parameter set\n",
+    "Once you're happy with a particular parameter set of your creation, you may want to contribute it to the PyBaMM parameter distribution, so\n",
+    "that it comes bundled with future release of PyBaMM.\n",
+    "This will require you to open a Pull Request on the PyBaMM repository, a process that is described here.\n",
+    "\n",
+    "A prerequisite is that you add you parameter set to the default parameter directory. You could manually copy the corresponding directory to your installation directory, but the command `pybamm_add_param` can do it for you:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# pybamm_add_parameter <dir> <chemistry> <compoent>\n",
+    "pybamm_add_parameter lithium-ion/cells/my_new_param_set lithium-ion cells"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's remove the `my_new_param_set` from the parameter search path to check that the parameter directory was added correctly to set of default parameters:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['/tmp',\n",
+       " '/tmp/virtualenvs/pybamm-venv/lib/python3.7/site-packages/pybamm/input/parameters']"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pybamm.PARAMETER_PATH.remove(\"lithium-ion/cells/my_new_param_set\")\n",
+    "pybamm.PARAMETER_PATH"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'a': 4,\n",
+       " 'b': 5,\n",
+       " 'c': 6}"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pybamm.ParameterValues(\"lithium-ion/cells/my_new_param_set/param_file.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you think you made a mistake, you can always delete a specific parameter set using `pybamm_rm_param`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "pybamm_rm_parameter -f lithium-ion/cells/my_new_param_set lithium-ion cells"
```

Comment:
got an error here (`pybamm_rm_parameter` not found)

Reply:
Hmm, did you reinstall pybamm? You'll need to if you want to have the command install and available in your `PATH`